pkg/frontend/admin_hive_syncset_resources.go

```diff
@@ -0,0 +1,69 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/ugorji/go/codec"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+)
+
+func (f *frontend) getAdminHiveClusterSync(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	log := ctx.Value(middleware.ContextKeyLog).(*logrus.Entry)
+	resourceId := strings.TrimPrefix(filepath.Dir(r.URL.Path), "/admin")
```

Comment:
nitpick:  Does it make sense to have an empty/malformed check on the `resourceId` ?

Reply:
Yes, this practice is consistent with the approach followed by other APIs as well, ensuring alignment and standardization across implementations. pkg/frontend/admin_openshiftcluster_drainnode.go

Reply:
suggested changes implemented